docs/source/howto/level1-csd.rst

```diff
@@ -0,0 +1,312 @@
+.. _tut-l1csd:
+
+Level 1 Cloud Storage Downloader
+================================
+
+**Downloading Landsat and Sentinel-2 Level-1 data from cloud services**
+
+This tutorial covers the process of querying and downloading Landsat and Sentinel-2 Level 1 data from cloud services (currently Google Cloud Storage).
+FORCE Level 1 CSD offers a high level of detail in requesting data and creates a consistent Level 1 data pool, as well as queue files that can be used for Level 2 processing.
+
+.. admonition:: Info
+
+   *This tutorial uses FORCE v. 3.5*
+
+FORCE Level 1 CSD makes use of the Landsat and Sentinel-2 metadata catalogues from Google Cloud Storage to find scenes matching the user's requirements and downloads them through `gsutil <https://cloud.google.com/storage/docs/gsutil>`_. A check for scenes already existing locally is performed and only new scenes are downloaded.
+
+
+Getting started
+---------------
+
+gsutil configuration
+""""""""""""""""""""
+
+Currently, Landsat data can be downloaded without a login (this will change at some point), but downloading Sentinel-2 data requires authentication.
+This is done through gsutil and only needs to be done once before the first run. The credentials will be stored in a file called ``.boto`` in the user's home directory (make sure to check the notes at the end of this document if you're running FORCE in Docker).
```

Comment:
Just some minor style comments here. 
I generally use one line for each sentence for readability in text editors.

Also, I like to have 2 lines after a special block like note, admonition, code, figure etc, as well as before a new heading.

Reply:
I like the idea of having a standardized formatting across the docs.
You may want to set up a very short document with formatting guidelines and maybe some hints on how to write/what tools to use. Afaik more people are in the process of contributing to the tutorials and it is much easier to implement a straight forward formatting in the beginning than changing it at a later point

Reply:
yep, I agree. This was the first time that this applied